### ISSUE TITLE
Error "empty error" when using DOT

### ISSUE THREAD

**Issue by utopos** [fuelen/ecto_erd#59]
Hi,

when using only DOT option (all others work fine)  I get the following error:

```elixir
** (Enum.EmptyError) empty error
    (elixir 1.17.3) lib/enum.ex:1904: anonymous fn/0 in Enum.max/1
    (ecto_erd 0.6.2) lib/ecto/erd/document/dot.ex:88: anonymous fn/2 in Ecto.ERD.Document.Dot.render_node/2
    (elixir 1.17.3) lib/enum.ex:1703: Enum."-map/2-lists^map/1-1-"/2
    (elixir 1.17.3) lib/map.ex:262: Map.new_from_enum/2
    (ecto_erd 0.6.2) lib/ecto/erd/document/dot.ex:82: Ecto.ERD.Document.Dot.render_node/2
    (elixir 1.17.3) lib/enum.ex:1804: anonymous fn/2 in Enum.map_join/3
    (elixir 1.17.3) lib/enum.ex:4408: Enum.map_intersperse_list/3
    (elixir 1.17.3) lib/enum.ex:4408: Enum.map_intersperse_list/3
```

**Comment by fuelen** [comment]
Thank you for the report.

As I understand, that's because you have a schema without any field, correct?

**Comment by utopos** [comment]
Hi Artur,

yes this might be the case indeed.

**Comment by fuelen** [comment]
Published v0.6.3
